iOS App Signer.xcodeproj/project.pbxproj

```diff
@@ -302,7 +302,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 14.5;
```

Comment:
what's the reason of this change?

Reply:
10.9 is an invalid deployment target and causes the build to fail. Plus the changes made previously to the certificate path only work on newer versions of macOS and will cause a breaking change on older versions.

Reply:
Changing the upload attract version is necessary to resolve the depreciation warning.

Reply:
> 10.9 is an invalid deployment target and causes the build to fail.

it's not a reason to set it to such a high one. 10.13 would be a good choice.

> Plus the changes made previously to the certificate path only work on newer versions of macOS and will cause a breaking change on older versions.

this is not true, check the change: https://github.com/DanTheMan827/ios-app-signer/commit/9ba3a18723d9599549ac2741512899da1e831496

Reply:
I have updated this in a new commit